lexers/c/ca65.go

```diff
@@ -0,0 +1,30 @@
+package c
+
+import (
+	"regexp"
+
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+var ca65AnalyserCommentRe = regexp.MustCompile(`(?m)^\s*;`)
+
+// Ca65 lexer.
+var Ca65 = internal.Register(MustNewLexer(
+	&Config{
+		Name:      "ca65 assembler",
+		Aliases:   []string{"ca65"},
+		Filenames: []string{"*.s"},
+		MimeTypes: []string{},
+	},
+	Rules{
+		"root": {},
+	},
+).SetAnalyser(func(text string) float32 {
+	// comments in GAS start with "#".
```

Comment:
I guess this is a mistake in pygments, right?
```suggestion
	// comments in GAS start with ";".
```

Reply:
I had the same feeling but actually in GAS you can comment either by `#` or `;` so it's explaining this difference.
https://github.com/pygments/pygments/blob/master/pygments/lexers/asm.py#L59